sdks/python/build.gradle

```diff
@@ -204,19 +204,30 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 
 // Before running this, you need to:
 //
-// 1. build the SDK container:
+// 1. Build the SDK container:
 //
 //    ./gradlew -p sdks/python/container docker
 //
-// 2. start a local JobService, for example, the Portable Flink runner
+// 2. Either a) or b)
+//  a) If you want the Job Server to run in a Docker container:
+//
+//    ./gradlew :b®eam-runners-flink_2.11-job-server-container docker
+//
+//  b) Otherwise, start a local JobService, for example, the Portable Flink runner
 //    (in a separate shell since it continues to run):
 //
 //    ./gradlew :beam-runners-flink_2.11-job-server:runShadow
 //
 // Then you can run this example:
 //
+//  Docker (2a): 
+//
 //    ./gradlew :beam-sdks-python:portableWordCount
 //
+//  Local JobService (2b):
+//
+//    ./gradlew :beam-sdks-python:portableWordCount --job_endpoint=localhost:8099
```

Comment:
Doesn't this require a project property?

Reply:
Ah yes.

Reply:
`Unknown command-line option '--job_endpoint`

Reply:
like this
``` 
if (project.hasProperty("flinkMasterUrl"))
    args += ["--flink-master-url=${project.property('flinkMasterUrl')}"]
```

Reply:
Updated to include a property.